src/style-library/components/pages/TypographyScreen.scss

```
@@ -94,3 +94,21 @@
 
 	align-items: center;
 }
+
+// The favorite button's busy stripes, restored for its Remove (tertiary) face. `wp-components`
+// flattens the background of a disabled secondary OR tertiary button, then re-asserts the busy
+// gradient over that for `.is-secondary.is-busy` alone — there is no tertiary counterpart, and the
+// generic `.components-button.is-busy` rule loses the specificity contest. So Add Favorite animates
+// while Remove Favorite sits perfectly still, which is the state this control most needs to show.
+// Only the background is clobbered; the animation itself survives, so re-declaring the gradient at a
+// weight the disabled rule cannot beat is the whole fix.
+//
```

Stylelint (17.14.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)